src/shared/components/CloudUpgradeNavBanner.tsx

```diff
@@ -49,23 +51,39 @@ const CloudUpgradeNavBanner: FC<StateProps> = ({inView, isRegionBeta}) => (
             </Heading>
           </Panel.Header>
           <Panel.Footer size={ComponentSize.ExtraSmall}>
-            <a
-              className="cf-button cf-button-md cf-button-primary cf-button-stretch cloud-upgrade-banner--button upgrade-payg--button__nav"
-              href={`${CLOUD_URL}${CLOUD_CHECKOUT_PATH}`}
-              target="_self"
-            >
-              Upgrade Now
-            </a>
+            {isFlagEnabled('unityCheckout') ? (
+              <Link
+                className="cf-button cf-button-md cf-button-primary cf-button-stretch cloud-upgrade-banner--button upgrade-payg--button__nav"
```

Comment:
should we create a component for this link so the styles can be defined in one place? Looks like we're duplicating this list of styles in a few places

Reply:
Not sure, where else is this being used outside of the CloudUpgradeNavBanner?

Reply:
Oh I was just thinking for the usage within this component. But I guess it's really only 2 places that all these styles are defined and 1 of those places will be removed once the `unityCheckout` flag is removed. So it's probably not worth changing